code_samples/recent_activity/src/Command/ActivityLogContextTestCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Event\MyFeatureEvent;
+use App\MyFeature\MyFeature;
+use Ibexa\Contracts\ActivityLog\ActivityLogServiceInterface;
+use Ibexa\Contracts\Core\Repository\ContentService;
+use Ibexa\Contracts\Core\Repository\ContentTypeService;
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\Content\LocationCreateStruct;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class ActivityLogContextTestCommand extends Command
+{
+    protected static $defaultName = 'doc:test:activity-log-context';
+
+    private ActivityLogServiceInterface $activityLogService;
+
+    private ContentService $contentService;
+
+    private ContentTypeService $contentTypeService;
+
+    private EventDispatcherInterface $eventDispatcher;
+
+    private PermissionResolver $permissionResolver;
+
+    private UserService $userService;
+
+    public function __construct(ActivityLogServiceInterface $activityLogService, ContentService $contentService, ContentTypeService $contentTypeService, EventDispatcherInterface $eventDispatcher, PermissionResolver $permissionResolver, UserService $userService)
+    {
+        parent::__construct(self::$defaultName);
+        $this->activityLogService = $activityLogService;
+        $this->contentService = $contentService;
+        $this->contentTypeService = $contentTypeService;
+        $this->eventDispatcher = $eventDispatcher;
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+    }
+
+    protected function configure(): void
+    {
+        $this->setDescription('Test activity log context usage')
```

Comment:
Description should be placed under `protected static $defaultDescription` instead.

Reply:
Done in 2d6bfc1b9e328ecfeedc2b9a7fc96ec6e36a12c9